### PR TITLE
Fixed NullpointerException of surface view

### PR DESCRIPTION
**Pull request by CircleCurve**

### CODE REVIEW

**Comment by CircleCurve** [comment]
I signed it!

File: packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java [hunk 1]
```diff
@@ -665,6 +665,9 @@ private void startPreview() throws CameraAccessException {
 
       Surface previewSurface = new Surface(surfaceTexture);
       surfaces.add(previewSurface);
+      if(surfaces == null || imageReader == null){
+        return;
+      }
```
Comment (Hixie) [inline packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: Why would they be null here? Are we doing something wrong in how we obtain them?

**Comment by Hixie** [comment]
@googlebot ping

**Comment by Hixie** [comment]
Looks like the cla bot may be confused with this PR. I recommend closing it and pushing a new PR.

**Comment by Hixie** [comment]
I'm going to close this PR since the bot is confused, just so that we don't accidentally commit the PR with the bot in this bad state. Please submit a new PR. Sorry about that.

**Comment by franzejr** [comment]
I was having this exactly issue and this solved the problem.

File: packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java (hunk 1 above)
Comment (franzejr) [reply packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: I also don't know yet why they would be null here. this is related to: https://github.com/flutter/flutter/issues/19595

Comment (franzejr) [reply packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: I think the `camera.open()` is not being called? 
https://github.com/CircleCurve/plugins/blob/1ceefbcc1f0e823bd2640fa3decbaa998b2be163/packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java#L95-L97

**Pull request closed without merging**